terraso_backend/tests/conftest.py

```diff
@@ -37,6 +37,16 @@ def users():
     return mixer.cycle(5).blend(User)
 
 
+@pytest.fixture
+def user(users):
```

Comment:
I pulled these fixture definitions up from another module, so that more tests can use them.